versions.tf

```
@@ -1,10 +1,10 @@
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = ">= 0.15.0"
 
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 3.0"
+      version = ">= 3.0, < 4.0"
```

I don't understand. I thought we weren't going to pin provider versions to major releases.

Also, this is more concise, no ?

```suggestion
      version = "~> 3.0"
```

---

Oh ok, so 2 separate releases interesting.

---

This is a temporary change to provide a stepping stone to the next version, where the version will be pinned at `>= 4.2`.